### DISCUSSION TITLE
New Name

### DISCUSSION THREAD

**Discussion by not-nef** [Futura-Py/FluxCalc#31] [Ideas]
Ok, we definetely need a new name because fluent python calculator is boring and too long

**Comment by not-nef** [comment]
what about flux? it starts with the same three letters as fluent and it sounds cool

**Comment by HuyHung1408** [comment]
it sounds good enough for me! Any other ideas?

**Reply by sumeshir26** [reply]
What about FlyCalc?

**Reply by not-nef** [reply]
good, but i think flux is cooler

**Reply by lg-nineoneninethreenine** [reply]
PyCalc? Flux seems disconnected to a calc app y'know...

**Reply by not-nef** [reply]
@HuyHung1408 you decide

**Reply by HuyHung1408** [reply]
All of them are good, but PyCalc seems to be the same name with some other repo, there's even the app called that name in Apple App Store. I choose Flycalc is good but I will go with Flux then. Flux Calculator?

**Reply by not-nef** [reply]
FluxCalc?

**Reply by lg-nineoneninethreenine** [reply]
Familiarity echoed when I said PyCalc, well ditch it, we shall try FluXCalc. Just a minor change to @not-nef's name, that seems better. Well at least to me.

**Comment by sumeshir26** [comment]
What about just flux?

**Reply by HuyHung1408** [reply]
Okay!
